docs/deployment/00-prereqs.yaml

```diff
@@ -69,6 +69,16 @@ rules:
   - create
   - get
   - update
+# Allow for listing/watching/getting Secrets resources.
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
+  - list
+  - update
```

Comment:
Do we need `update` here?

Reply:
Yes, to keep track if the secret has been reflected or not. Offending code [here](https://github.com/mesosphere/dklb/blob/master/pkg/secrets_reflector/secrets_reflector.go#L142).